python/lang/security/deserialization/pickle.yaml

```diff
@@ -0,0 +1,48 @@
+rules:
+  - id: avoid-pickle 
+    metadata:
+      owasp: "A8: Insecure Deserialization"
```

Comment:
Can we add CWEs too?

CWE-502: Deserialization of Untrusted Data
https://cwe.mitre.org/data/definitions/502.html

`cwe: "CWE-502: Deserialization of Untrusted Data"`